spec/rubocop/cop/rails/attribute_default_block_value_spec.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::AttributeDefaultBlockValue do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+  let(:message) { 'Pass method in a block to `:default` option.' }
+
+  context 'when `:default` option is last' do
+    it('disallows method') do
```

Comment:
Can you remove parentheses for `it` of overall of the PR?

```suggestion
    it 'disallows method' do
```